be/src/vec/common/allocator.h

```diff
@@ -120,6 +120,43 @@ static constexpr size_t MALLOC_MIN_ALIGNMENT = 8;
         }                                                           \
     } while (0)
 
+#define SYS_MEMORY_CHECK(size)                                                                     \
```

Comment:
This should be a method. Not a macro.
Should avoid use macro if it is possible because the code is not easy to read.

Reply:
done